README.md

```diff
@@ -76,6 +76,12 @@ $(go env GOPATH)/bin/mkcert
 
 or use [the pre-built binaries](https://github.com/FiloSottile/mkcert/releases).
 
+To download the latest version the pre-built binaries you can use this script:
+
+```bash
+bash <(wget -qO- https://git.io/mkcert) --install-dir=/usr/local/bin
```

Comment:
I'm not a fan of `curl ... | bash` type installs as they open up a whole field of possible compromise vectors. Plus Github doesn't own `git.io`.

Reply:
may be a verification gpg signature will solve the problem of trust  to `git.io`

Reply:
ok, I updated docs